src/infrastructure/transport/fetch.transport.ts

```diff
@@ -51,14 +51,41 @@ export default class FetchTransport {
     return this.parseResponse(response, endpoint);
   }
 
+  /**
+   * Gets specific resource in binary
+   * @param endpoint - API endpoint
+   * @param data - data to be sent url encoded
+   */
+  public async getBlob(endpoint: string, data?: JSONValue): Promise<Blob> {
+    const resourceUrl = new URL(this.baseUrl + endpoint);
+
+    if (data !== undefined) {
+      resourceUrl.search = new URLSearchParams(data as Record<string, string>).toString();
```

Comment:
Change JSONvalue to Record<string, string> or this bird will come for you

![image](https://github.com/codex-team/notes.web/assets/40016207/f2498106-476a-4314-abb8-062f9a062ba6)

Reply:
done